### ISSUE TITLE
Replicating the results in the paper

### ISSUE THREAD

**Issue by Jianbo-Lab** [williamleif/GraphSAGE#32]
Hi,

In the paper, the F1 on the PPI data set for the supervised setting is 0.598 (for GraphSAGE-mean). The default hyperparameters yield an F1 of around 0.576 on the full data set. 

Can I know whether the difference is due to the different hyperparameter choices or just some randomness from stochastic sampling? If the former, please share with me the concrete choice of parameters if possible. 

Thanks!

Jianbo

**Comment by buptjz** [comment]
The default hyper-parameters yield an average f1_micro of around 0.517.

> Optimization Finished!
Full validation stats: loss= 0.51058 f1_micro= 0.51657 f1_macro= 0.31484 time= 0.11630
Writing test set stats to file (don't peak!)


I just do as the following:
1. clone this project.
2. Change the line  "FROM tensorflow/tensorflow:1.3.0-gpu" in Dockerfile.gpu. (I also tried 1.5.0-gpu version)
3. build & run in docker with example_supervised.sh

**Comment by yangsiran** [comment]
Hi,

In my case of experiment, the default parameters (lr=0.01, neighbor samples of 10 x 25, hidden layer dim=128) running with 10 epochs yielded micro F1 score in test set of 0.57455. While the same parameters running with 20 epochs got 0.5984.
